flax/linen/fp8_ops.py

```diff
@@ -277,19 +383,18 @@ def __call__(self, *args, **kwargs):
     # namely, the computation data type.
     comp_dtype = k.dtype
     x = jnp.asarray(x, comp_dtype)
-
-    x_qdq = in_qdq(
-      comp_dtype, x, self.input_scale.value, self.input_amax_history.value
-    )
-    k_qdq = in_qdq(
-      comp_dtype, k, self.kernel_scale.value, self.kernel_amax_history.value
-    )
-    y_qdq = dot_general_with_precision(x_qdq, k_qdq, dimension_numbers)  # type: ignore
-    y = out_qdq(
-      comp_dtype,
-      y_qdq,
-      self.output_grad_scale.value,
-      self.output_grad_amax_history.value,
+    y = _q_dot_dq(
```

Comment:
Let's call it `q_dot_dq` because I guess we would like to use it in other modules like in praxis fp8 einsum.